.gitignore

```diff
@@ -1 +1,7 @@
-node_modules/
\ No newline at end of file
+node_modules/
+hopital/build/
+hopital/bdd/
+hopital/nbproject/
+hopital/test/
```

Comment:
à garder non ?

Reply:
le dossier test est vide, le dossier bdd c'est ma base de données et nbproject contient la config de mon ide

Reply:
je pense que seul `nbproject/private/` est à ignorer
cf https://gist.github.com/chhh/4961200
cf https://stackoverflow.com/questions/2586097/what-should-be-in-gitignore-file-for-a-netbeans-java-project

Reply:
Le dossier nbproject/ contient seulement des fichiers de configuration du projet selon l'ide

Reply:
ok pour ignorer test/

Reply:
Le dossier test/ correspond à la partie des tests sur le code (il n'a aucune utilité ici)